samples/ResourceGroup/copy-resourceGroup-tag/README.md

```diff
@@ -0,0 +1,34 @@
+# Copy resource group tag to resource
```

Comment:
[naming convention](https://github.com/Azure/azure-policy/tree/master/1-contribution-guide#files-folders-and-naming-conventions): all folder names should be entirely lowercase:
copy-resource**g**roup-tag